README.md

```diff
@@ -94,14 +94,14 @@ module.exports = lambdaCfn.build({
 });
 ```
 
-If you'd like to create a Lambda function that runs on the older Node.js 4.3 runtime with a memory size of 256 MB and a timeout of 120 seconds:
+If you'd like to create a Lambda function that runs on the newer Node.js 10.5 runtime with a memory size of 256 MB and a timeout of 120 seconds:
 
 ```js
 const lambdaCfn = require('@mapbox/lambda-cfn');
 
 module.exports = lambdaCfn.build({
   name: 'myFunction',
-  runtime: 'nodejs6.10',
+  runtime: 'nodejs10.5',
```

Comment:
I think this is a typo? should be `nodejs10.x`?

Reply:
Yup, updating now.